.gitignore

```diff
@@ -33,3 +33,6 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+# Ignore application configuration
+/config/application.yml
```

Comment:
This one's surprising, that's usually included in the repo and it contains bindings to `ENV`s 🤔 

What was the idea here?

Reply:
Good to know! This one is on Figaro - [it's how it setups by default](https://github.com/laserlemon/figaro#getting-started). Would it be more beneficial if I tried to do it some other way?

Reply:
oh God, now I see it - thanks Kate, TIL.

In that case it's OK and good to go 👌 I thought `figaro` works similar to `dotenv` - using the `.env` file. `figaro`s README proven me wrong :)

Good to go then!

Reply:
Almost! Because still, even though I can access that env via rails console (proving this setup is ok), database.yml has some issues with it, meaning no db can be created.

Reply:
Now everything works as it should, I think it's ready to merge 👌🏻

Reply:
It almost is - we can remove the entry about `config/application.yml` from `.gitignore` - no need for any leftover mess